{{cookiecutter.project_slug}}/docs/index.rst

```diff
@@ -15,7 +15,9 @@ Contents:
    installation
    usage
    contributing
-   authors
+   {% if cookiecutter.create_author_file == 'y' -%}
+      authors
```

Comment:
I think `authors` needs to be unindented so that the Sphinx docs' table of contents renders properly when building the Sphinx docs.